cmd/mdmctl/apply.go

```diff
@@ -68,9 +78,90 @@ Examples:
 
 func (cmd *applyCommand) applyBlueprint(args []string) error {
 	flagset := flag.NewFlagSet("blueprints", flag.ExitOnError)
+	var (
+		flBlueprintPath    = flagset.String("f", "", "filename of blueprint JSON to apply")
+		flNewBlueprintPath = flagset.String("n", "", "filename of new blueprint JSON to create")
```

Comment:
It's a helper utility to quickly generate a new blueprint (which you can then edit to make a real one). It generetes a UUID, puts in the server URL, etc. — bunch of details the server already knows so a user can get on with managing things rather than sorting out what fields and format the blueprint JSON needs to be in.

I'm open to moving it or change semantics or whatever. I just think its helpful. Perhaps when other capabilities are added it will be unnecessary.